systests/databinding/pom.xml

```diff
@@ -165,7 +165,7 @@
             <plugin>
                 <groupId>org.jibx</groupId>
                 <artifactId>maven-jibx-plugin</artifactId>
-                <version>${cxf.jibx.version}</version>
+                <version>1.3.1</version>
```

Comment:
Hey Daniel!

I think because the version specified by the variable is needed to make the code compile and produce the correct artifacts (the bindings), but the plugin needs to be a newer version to work with JDK 1.8 and modern maven. 

Keep in mind I have this branch marked WIP as I sort bugs out. The goal is to fix all the known CVEs in 2.6 and make it buildable without having to maintain a separate environment.